libs/git/repository.go

```diff
@@ -44,12 +54,11 @@ type Repository struct {
 
 // Root returns the absolute path to the repository root.
 func (r *Repository) Root() string {
```

Comment:
Let's keep the `Root()` reserved for the repo root, and expose checkoutDir via a new method like `CheckoutDir`?

Right now `Root()` is also used to set the `bundle_root_path` git metadata in the `metadata_flle_file` we set for jobs metadata which is then used to provide the "open in repos" functionality for a job deployed from DABs, which would break because now bundle_root_path is no longer from the root of the repo, but instead is from the worktree root.

Also, semantically just makes sense for `Root()` to return the repo "root".

Reply:
Renamed back to `rootDir`. I wasn't 100% happy with "root" because of the ~ambiguity considering worktrees. But not 100% happy with "checkout" either.